srcpkgs/oils/template

```diff
@@ -1,18 +1,19 @@
-# Template file for 'oil'
-pkgname=oil
+# Template file for 'oils'
+pkgname=oils
 version=0.22.0
 revision=1
 build_style=configure
 configure_args="--prefix=/usr"
 hostmakedepends="$(vopt_if readline readline-devel)"
 makedepends="$hostmakedepends"
-short_desc="Oil, a new unix shell"
+short_desc="Oils, a new unix shell"
 maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
 license="Apache-2.0"
 homepage="https://www.oilshell.org"
 distfiles="https://www.oilshell.org/download/oil-${version}.tar.xz"
 checksum=452e7fd428b686aa752cfeb9be2b94f9fcf7ba9ab22e0ae570a87cdcf782242e
-register_shell="/usr/bin/osh /usr/bin/oil"
+replaces="oil<=0.23.0"
```

Comment:
should be a transitional subpackage not replaces

Reply:
I think both replace and a transitional subpackage are appropriate here.

Reply:
I consider it more appropriate to replace, than transitional subpackage, only the names were changed, it is still the same model.

Reply:
without the transitional package, users would be stuck on the old package forever if they don't know to install the other one

Reply:
Done!